src/kzg10/mod.rs

```diff
@@ -158,31 +158,25 @@ impl<E: PairingEngine> KZG10<E> {
     }
 
     /// Compute witness polynomial.
+    /// Originally the witness polynomial W(x) is computed as the quotient of f(x) - f(z) / x - z
+    /// One can observe that the quotient does not change when f(z) is changed because
+    /// f(z) is the remainder term. We can therefore, remove f(z) when computing the witness
```

Comment:
```suggestion
        /// 
        /// Originally the witness polynomial w(x) is computed as the quotient of (p(x) - p(z)) / (x - z)
    /// One can observe that this quotient does not change with z is changed because
    /// p(z) is the remainder term. We can therefore omit p(z) when computing the quotient
```